src/YobotBrokerV2.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.11;
+
+struct Order { // wei per token is: amountDeposited / quantity
```

Comment:
how does wei per token work? i.e. if weiDeposited > mint price + gas price, where does the unspent $ go?

Reply:
assuming whichever bot picks up the tx gets to keep the additional unspent $, right?

(do you think you could point me twds the code in our yobot-searcher repo where this mechanism is built? cc @abigger87 )

Reply:
yea so the bot will get the unspent wei

Reply:
Would be neat if the ui could be like enter price that's roughly mint price + gas price